scripts/generate.sh

```diff
@@ -109,6 +109,10 @@ apt-get update && apt-get -y install elasticsearch=5.5.3 && \
 echo "RUN pip install sh && \
 apt-get -y install postgresql postgresql-contrib"
```

Comment:
Could you add to this set of lines, instead of echoing a new line for each command?

Something like:
```
echo "RUN pip install sh && \
apt-get -y install postgresql postgresql-contrib && \
mkdir -p /usr/local/pgsql/data && \
chown -R postgres:postgres /usr/local/pgsql && \
su -c '/usr/lib/postgresql/9.5/bin/initdb -D /usr/local/pgsql/data' postgres"
```

Reply:
Done